WordPress/Classes/ViewRelated/Likes/LikesListController.swift

```diff
@@ -208,12 +249,12 @@ class LikesListController: NSObject {
 extension LikesListController: UITableViewDataSource, UITableViewDelegate {
 
     func numberOfSections(in tableView: UITableView) -> Int {
-        return Constants.numberOfSections
+        return showingNotificationLikes ? 2 : 1
```

Comment:
If we wanted to get fancy we could encapsulate the section numbers, and the userSectionIndex, in structs that are assigned during init.  Just a thought, and totally not necessary.

Reply:
Ah, yea, true. I'll take a look later. Thanks!